src/memex/storage.py

```diff
@@ -11,6 +11,7 @@
 
 from pyramid import i18n
 
+from memex import GROUPFINDER_KEY
```

Comment:
This is ok, but I think it would be better to encapsulate the registry storage within a module dedicated to that task, say `memex.groups`, and provide a helper function like `memex.groups.find(...)` which pulls the relevant thing out of the registry.